lib/dataspec/src/lib.rs

```diff
@@ -0,0 +1,77 @@
+use serde::{Deserialize, Serialize};
```

Comment:
Another suggestion: a small doc comment here explaining what the library is would be helpful to others.